jest.config.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
   moduleFileExtensions: ['ts', 'js', 'json', 'node'],
   coveragePathIgnorePatterns: ['node_modules', 'testHelpers'],
   globalSetup: '<rootDir>/test/integration/environment/setup.js',
+  testTimeout: 15000,
```

Comment:
are these seconds? Seems like a lot

unit tests time in red, should we check that?

![image](https://github.com/IQSS/dataverse-client-javascript/assets/23359572/b61812c5-0267-4309-bf10-6901be920d6c)

Reply:
Integration tests can take some time, as multiple Dataverse API calls are made in the integration test setup methods to configure certain scenarios and preconditions in the Dataverse test container instance. 

Initially, I established this timeout by test, where I found it necessary. As I repeated it a lot, I decided to set it globally. Note that this timeout is only set when running the integration test suite (npm test and npm test:integration). When we want to run only the unit tests (by running npm test:unit), the timeout is the default one.

I'm not worried about the timing of the unit tests at the moment (There are many tests per file). I'll keep that in mind more from now, but I think it is not an issue or blocker for this PR.